doc/windows/package-manager/winget/returnCodes.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Exit Codes
+description: WinGet return codes and their meanings
+ms.date: 05/02/2023
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Return Codes
+
+## General Errors
+
+| Hex | Decimal | Description |
+|-------------|-------------|-------------| 
+| 0x8A150001 | -1978335231 | Internal Error
+| 0x8A150002 | -1978335230 | Invalid command line arguments
+| 0x8A150003 | -1978335229 | Executing command failed
+| 0x8A150004 | -1978335228 | Opening manifest failed
+| 0x8A150005 | -1978335227 | Cancellation signal received
+| 0x8A150006 | -1978335226 | Running ShellExecute failed
+| 0x8A150007 | -1978335225 | Cannot process manifest. The manifest version is higher than supported. Please update the client.
+| 0x8A150008 | -1978335224 | Downloading installer failed
+| 0x8A150009 | -1978335223 | Cannot write to index | it is a higher schema version
```

Comment:
The `|` looks like it could mess things up with the table

Reply:
Good catch